taxonomies/ingredients.txt

```diff
@@ -52144,6 +52144,15 @@ ciqual_food_code:en:13004
 ciqual_food_name:en:Avocado, pulp, raw
 ciqual_food_name:fr:Avocat, pulpe, cru
 
+<en:avocado
+en:Hass avocado
```

Comment:
I t would be good to add the plurals as well, especially as just adding "s" as we do for matching won't work in that case.

```suggestion
en:Hass avocado, Hass avocadoes
```